### ISSUE TITLE
ReferenceError: console is not defined

### ISSUE THREAD

**Issue by KyleAMathews** [petehunt/webpack-require#4]
Was playing around with this a bit just now. I could require my webpack module just fine but when I went to render it (server-side), I got this error:

```
ReferenceError: console is not defined
    at warning (eval at <anonymous> (undefined:739:2), <anonymous>:48:7)
    at Constructor (eval at <anonymous> (undefined:967:2), <anonymous>:834:50)
    at /root/RelateRocket/page-server/lib/refactored-page-handler.coffee:36:38
    at /root/RelateRocket/page-server/lib/webpack-require.coffee:10:12
```

Same code works just fine outside of `webpack-require` so not sure what's happening.

**Comment by petehunt** [comment]
You were calling `console.log()` in your code somewhere, right?

**Comment by petehunt** [comment]
should be fixed in 0.0.10
